README.rst

```diff
@@ -41,6 +41,12 @@ Optional: install example webclient plugins:
 
 Now restart OMERO.web as normal.
 
+**Warning**:
+
+OMERO.webtest version 0.2.x or newer is meant to be installed into OMERO.web **5.2.6 or newer**. This is due to Django Framework compatibility and a package reorganization required to distribute the application using Python Package Manager PyPi.
+
+
```

Comment:
I couldn't find my comment either! I am struggling to find any comments since e-mails don't have links to the comments - only link to the PR itself.
I even tried to find the change that @jburel made in response to my comment, but it seems that has got wiped?

https://github.com/ome/omero-figure/commit/ba48b0a and https://github.com/ome/omero-figure/commit/387cea3 are no-longer under
https://github.com/ome/omero-figure/pull/181/commits

@jburel Did you force push?